docs/src/pages/components/masonry/ImageMasonry.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function ImageMasonry() {
+  return (
+    <Masonry
+      columns={{ xs: 3, sm: 5 }}
+      spacing={{ xs: 1, sm: 1.5 }}
+      sx={{ height: 300 }}
+    >
+      {itemData.map((item) => (
+        <MasonryItem key={item.img}>
+          <img
+            srcSet={`${item.img}?w=164&h=164&auto=format 1x,
```

Comment:
With the above columns, I believe the max image width is 206px (please double check). No need to specify height.